stage/pulumi_run_recorder_test.go

```diff
@@ -0,0 +1,44 @@
+package stage
+
+import (
+	"context"
+	"testing"
```

Comment:
You can import "github.com/stretchr/testify/assert" and "github.com/stretchr/testify/require" here and use assert/require libraries below. You can check client_test.go as an example.